### PR TITLE
Ability to search by location

### PR DESCRIPTION
**Pull request by gpineiro-vivid**

### CODE REVIEW

**Comment by dhlevi** [comment]
@gpineiro-vivid Some minor conflicts to correct after a merge from a previous PR. Most is pretty straightforward but there is a change to a function that I'd rather you verify as it changes it quite a bit!

File: client/wfnews-war/src/main/angular/src/app/components/active-wildfire-map/active-wildfire-map.component.html [hunk 1]
```diff
@@ -25,13 +36,11 @@ <h2 class="wildfire-count">
                 </button>
             </div>
             <div class="firecentre-filter">
-                <select appWFSingleSelect appWFPlaceholder="Select a fire centre..."
-                    [options]=""
-                    [selected]="">
+                <select appWFSingleSelect appWFPlaceholder="Select a fire centre..." [options]="" [selected]="">
```
Comment (dhlevi) [inline client/wfnews-war/src/main/angular/src/app/components/active-wildfire-map/active-wildfire-map.component.html:39]: We have removed the Fire Centre filter from the UI. Not really a part of this ticket though so we can remove this as a separate task

File: client/wfnews-war/src/main/angular/src/app/components/active-wildfire-map/active-wildfire-map.component.ts [hunk 2]
```diff
@@ -54,18 +64,27 @@ export class ActiveWildfireMapComponent implements OnInit, OnChanges {
...
             const mapConfig = [];
-            // this.checkMapServiceStatus()
-            // .then( ( mapServiceStatus ) => {
-            // console.log(mapServiceStatus)
-            // this.wfnewsMapService.mapServiceStatus = mapServiceStatus
-
+            
             this.mapConfigService.getMapConfig() // this.applicationConfig.device )
                 // .then((config) => {
                 //     mapConfig.push(config)
```
Comment (dhlevi) [inline client/wfnews-war/src/main/angular/src/app/components/active-wildfire-map/active-wildfire-map.component.ts:90]: If it isn't used, we can remove the commented out stuff

**Review by dhlevi — APPROVED** [review]

File: client/wfnews-war/src/main/angular/src/app/components/active-wildfire-map/active-wildfire-map.component.html (hunk 1 above)
Comment (gpineiro-vivid) [reply client/wfnews-war/src/main/angular/src/app/components/active-wildfire-map/active-wildfire-map.component.html:39]: Yeah, I just grabbed the UI that Lucas leave me in another branch. Makes sense.

File: client/wfnews-war/src/main/angular/src/app/components/active-wildfire-map/active-wildfire-map.component.ts (hunk 2 above)
Comment (gpineiro-vivid) [reply client/wfnews-war/src/main/angular/src/app/components/active-wildfire-map/active-wildfire-map.component.ts:90]: Sure, I wasn't sure to delete the comments, but I'll go ahead and start to remove those.

**Comment by gpineiro-vivid** [comment]
@dhlevi branch updated

**Pull request merged**